README.md

```diff
@@ -104,11 +104,26 @@ query(
 ```js
 query(
   fromDB(db),
-  and(equal(db.seekType, 'post', 'type')),
+  and(equal(seekType, 'post', 'type')),
   toCallback((err, msgs) => {
     console.log('There are ' + msgs.length + ' messages of type "post"')
   })
 )
+
+// The `seekType` function takes a buffer and uses `bipf` APIs to search for
+// the fields we want. Note that `~p` returns true for both `0` and `1` but
+// false for `-1`:
```

Comment:
Why not just use (p > -1)
Everyone knows what that means, and it's the and saves one character over (!~p)

Reply:
Wow middle of the night comments have shonky grammer

Reply:
There is no perf difference between `if(~p) return` and `if (p == -1) return`. So yeah probably go for the more readable version.